docs/topics/note-booster-unvailable-oso.adoc

```diff
@@ -1 +1,3 @@
-IMPORTANT: This booster is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
\ No newline at end of file
+//IMPORTANT: This booster is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
+*Limitation:* Run this booster on a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
```

Comment:
Nit: multiple sentences on single line :-)

Reply:
Ah nevermind, I see it's on multiple places, so the "one line = one sentence" convention probably doesn't apply in this repo. Sorry about the noise.